packages/fyllut-backend/src/middleware/tryCatch.ts

```diff
@@ -1,9 +1,14 @@
 import { NextFunction, Request, Response } from 'express';
 import { htmlResponseError } from '../utils/errorHandling';
+import { isValidPath } from '../utils/url';
 
 const tryCatch =
   (fn: (req: Request, res: Response, next: NextFunction) => void) =>
   async (req: Request, res: Response, next: NextFunction) => {
+    const formPath = req.params?.form;
```

Comment:
Synes kanskje det er litt rart å sjekke dette i denne middleware'n, for den brukes også til flere andre endepunkter som ikke sender formPath i det hele tatt.

`req.params.form` som det sjekkes på her vil kun slå til for kall til `/api/translations/:form`, og siden du har lagt inn validerering i TranslationsService.loadTranslation tenker jeg det kan fjernes fra middleware her.